python/pyspark/sql/column.py

```diff
@@ -934,8 +1064,25 @@ def between(
 
         .. versionadded:: 1.3.0
 
+        Parameters
+        ----------
+        lowerBound : :class:`Column` or :class:`LiteralType`
```

Comment:
I think it is better to avoid referencing typing aliases in the docs.

Reply:
oh, I just find that `LiteralType`/`DateTimeLiteral`/`DecimalLiteral` are typing aliases. Do you mean that we need use origin type here?

Reply:
Updated. Use origin type in latest commit.